types/matomo-tracker-react-native/index.d.ts

```diff
@@ -0,0 +1,94 @@
+// Type definitions for matomo-tracker-react-native 0.3
+// Project: https://github.com/donni106/matomo-tracker-react-native
+// Definitions by: Zeynep Ece Ergin <https://github.com/zecergin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.4
+
+import * as React from 'react';
+
+export interface AppStart {
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export interface Action {
+  name: string;
+  userInfo?:
+  {
+    uid?: string
+  };
+}
+
+export interface Event {
+  category: string;
+  action: string;
+  name?: string;
+  value?: number;
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export interface SiteSearch {
+  keyword: string;
+  category?: string;
+  count?: number;
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export interface Link {
+  link: string;
+  url?: string;
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export interface Download {
+  download: string;
+  url?: string;
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export function useMatomo(): {
+  trackAppStart: (params: AppStart) => undefined | Promise<Response>,
```

Comment:
just a comment, the default is other way around, eg..g when using VScode + Copilot, or more common sense